shared/index.native.js

```diff
@@ -45,8 +49,8 @@ class Keybase extends Component {
 
   render () {
     return (
-      <Provider store={store}>
-        <Nav />
+      <Provider store={this.store}>
+        {dumbSheetOnly ? <DumbSheet /> : <Nav />}
```

Comment:
Should we remove the [`dumbFullscreen`](https://github.com/keybase/client/blob/6e063675d6289c9ed3cf0ff391e35cf005036a96/shared/nav.android.js#L106) config since we now have this?